beaker/ext/redisclusternm.py

```diff
@@ -0,0 +1,168 @@
+import os
+import threading
+import time
+import pickle
+
+try:
+    import redis
+except ImportError:
+    redis = None
+
+from beaker.container import NamespaceManager
+from beaker.synchronization import SynchronizerImpl
+from beaker.util import SyncDict, machine_identifier
+from beaker.crypto.util import sha1
+from beaker._compat import string_type, PY2
+
+
+class RedisClusterNamespaceManager(NamespaceManager):
```

Comment:
Can't this inherit from `RedisNamespaceManager` and only override `__init__`? What's the benefit of reimplementing all other methods?

Actually, is there a reason why this can't be a simple option to the `RedisNamespaceManager` like `cluster=true`? Or even `if "," in urls`?

Reply:
There is no point at the moment. Indeed inheriting from `RedisNamespaceManager` could be the best option to prevent clutter in the `RedisNamespaceManager ` file ans to not reimplementing all.

I did not do that before because I couldn't see it elsewhere in the code and I was not sure if it was ok that some drivers depend on other's code.

Will do that

Reply:
Fixed, using inheritance and overrides to prevent reimplementing identical code.